### PR TITLE
Feature/sepolia deployment

### PR DESCRIPTION
**Pull request by tomrpl**
Do we want us to deploy the bundler on sepolia?

fixes INTEG-549

**Morpho Blue:** [0xd011EE229E7459ba1ddd22631eF7bF528d424A14](https://sepolia.etherscan.io/address/0xd011ee229e7459ba1ddd22631ef7bf528d424a14#readContract)

**AdaptiveCurveIrm:** [0x8C5dDCD3F601c91D1BF51c8ec26066010ACAbA7c](https://sepolia.etherscan.io/address/0x8C5dDCD3F601c91D1BF51c8ec26066010ACAbA7c#code)

**LLTV:**
[0, 385000000000000000, 625000000000000000, 770000000000000000, 860000000000000000, 915000000000000000, 945000000000000000, 965000000000000000, 980000000000000000] 

**Oracles:**
WETH/USDC Oracle: [0xaF02D46ADA7bae6180Ac2034C897a44Ac11397b2](https://sepolia.etherscan.io/address/0xaF02D46ADA7bae6180Ac2034C897a44Ac11397b2#readContract)

**Markets:**

**WETH/USDC (86%, MorphoChainlinkOracleV2, AdaptiveCurve IRM):** 0x712d4483aed77381acb0cd800813a9a1df61b4f6f3884024828a28e53e8471ee

**WETH/USDC (91.5%, MorphoChainlinkOracleV2, AdaptiveCurve IRM):** 
0xe7647bb54063bd2143cc907edc7a9fadf560991bc70e781121ac96bfe60380c5

**Idle USDC:**
0xfa7a1f738f36416666f6eb8259143b4161f04aa9de5b0450cd43d679c58a3994

**MetaMorpho Factory:** [0x2eB458F483Ae48AF2e79aFC21164Bc75eab3a8d5](https://sepolia.etherscan.io/address/0x2eB458F483Ae48AF2e79aFC21164Bc75eab3a8d5#code)

### CODE REVIEW

File: broadcast/DeployMetaMorphoFactory.sol/11155111/run-latest.json
Comment (Rubilmax) [inline broadcast/DeployMetaMorphoFactory.sol/11155111/run-latest.json:1]: It should also have archived the previous latest run into another file (but it seems it didn't unfortunately). Tant pis!

File: script/metamorpho-factory/verify.sh
Comment (Rubilmax) [inline script/metamorpho-factory/verify.sh:1]: This is the verify script that should not get committed

File: script/morpho/verify.sh
Comment (Rubilmax) [inline script/morpho/verify.sh:1]: Verify script to not commit (don't delete it but revert the changes)

File: script/oracles/DeployOracles.sol [hunk 1]
```diff
@@ -34,33 +38,29 @@ contract DeployOracle is ConfiguredScript {
...
                     ),
                     oracleConfig.salt
                 )
             );
-
-            require(address(oracle.VAULT()) == vault, "unexpected vault");
-            require(
-                oracle.VAULT_CONVERSION_SAMPLE() == oracleConfig.vaultConversionSample,
-                "unexpected vault conversion sample"
-            );
```
Comment (Rubilmax) [inline script/oracles/DeployOracles.sol:63]: can't this be translated with `BASE_VAULT`, `BASE_VAULT_CONVERSION_SAMPLE` etc?

File: script/oracles/verify.sh
Comment (Rubilmax) [inline script/oracles/verify.sh:1]: Verify scrip to not commit

**Review by Rubilmax — CHANGES_REQUESTED** [review]

File: broadcast/DeployMetaMorphoFactory.sol/11155111/run-latest.json
Comment (tomrpl) [reply broadcast/DeployMetaMorphoFactory.sol/11155111/run-latest.json:1]: it has, but I didn't pushed them.

SHould I?

image

**Review by Rubilmax — COMMENTED** [review]
almost complete

**Review by Rubilmax — APPROVED** [review]
